e2e/paths/enterChildren_test.js

```diff
@@ -52,7 +52,7 @@ Scenario('completing entering child information in the c110a application', (I, e
   enterChildrenPage.enterSocialWorkerDetails();
   enterChildrenPage.defineChildAdditionalNeeds();
   enterChildrenPage.defineContactDetailsVisibility();
-  enterChildrenPage.defineAbilityToTakePartInProceedings();
+  enterChildrenPage.enterLitigationIssues('No');
```

Comment:
Shall we do Yes from time to time to exercise SHOW / HIDE logic?

Reply:
Yes lets do one for each child, respondent and other person.